scripts/eurostat/health_determinants/alcohol_consumption/process_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for EuroStat BMI process script.
+"""
+
+import os
+import unittest
+import sys
+import tempfile
+# module_dir is the path to where this test is running from.
+MODULE_DIR = os.path.dirname(__file__)
+sys.path.insert(0, MODULE_DIR)
+# pylint: disable=wrong-import-position
+from process import EuroStatAlcoholConsumption
+# pylint: enable=wrong-import-position
+
+TEST_DATASET_DIR = os.path.join(MODULE_DIR, "test_data", "datasets")
+EXPECTED_FILES_DIR = os.path.join(MODULE_DIR, "test_data", "expected_files")
+OUTPUT_DATA_DIR = os.path.join(MODULE_DIR, "test_output_data")
+
+
+class TestProcess(unittest.TestCase):
+    """
+    TestPreprocess is inherting unittest class
+    properties which further requried for unit testing.
+    The test will be conducted for EuroStat BMI Sample Datasets,
+    It will be generating CSV, MCF and TMCF files based on the sample input.
+    Comparing the data with the expected files.
+    """
+    test_data_files = [
+        'hlth_ehis_al1e.tsv', 'hlth_ehis_al1i.tsv', 'hlth_ehis_al1u.tsv',
+        'hlth_ehis_al3e.tsv', 'hlth_ehis_al3i.tsv', 'hlth_ehis_al3u.tsv',
+        'hlth_ehis_al2e.tsv', 'hlth_ehis_al2i.tsv', 'hlth_ehis_al2u.tsv',
+        'hlth_ehis_al1b.tsv', 'hlth_ehis_al1c.tsv', 'hlth_ehis_de10.tsv',
+        'hlth_ehis_de6.tsv'
+    ]
+    ip_data = [
+        os.path.join(TEST_DATASET_DIR, file_name)
+        for file_name in test_data_files
+    ]
+
+    def __init__(self, methodName: str = ...) -> None:
+        super().__init__(methodName)
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            cleaned_csv_file_path = os.path.join(tmp_dir, "data.csv")
+            mcf_file_path = os.path.join(tmp_dir, "test_census.mcf")
+            tmcf_file_path = os.path.join(tmp_dir, "test_census.tmcf")
```

Comment:
Can you please try making these to constants and protected; if they are only one time assignment.

Reply:
Converted to constants and protected.